<script>

    try {

        alert('Начало блока try'); // (1)

        // .. код без ошибок

        alert('Конец блока try');  // (2)

    } catch (e) {

        alert('Блок catch не получит управление, так как нет ошибок');

    }

    alert('Потом код продлжит выполнение...');

</script>